resources/gemini-1b/provisioner.tf

```diff
@@ -28,8 +28,9 @@ resource "null_resource" "setup_nodes" {
     host = digitalocean_droplet.gemini-1b[count.index].ipv4_address
     user = "root"
     type = "ssh"
-    agent = true
-    agent_identity = var.ssh_identity
+    agent = false
+#    agent_identity = var.ssh_identity
+    private_key = var.alexey2_do_private_key
```

Comment:
Why this change here and below?

Reply:
its used for tf cloud

Reply:
I'd like a bit longer explanation than that. Why does it have to include your private key in variables?